src/DepartmentService.Models.Dto/Configurations/RabbitMqConfig.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Configurations;
+using LT.DigitalOffice.Models.Broker.Common;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
+using LT.DigitalOffice.Models.Broker.Requests.Message;
+using LT.DigitalOffice.Models.Broker.Requests.Project;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+
+namespace LT.DigitalOffice.DepartmentService.Models.Dto.Configuration
+{
+  public class RabbitMqConfig : BaseRabbitMqConfig
+  {
+    public string GetDepartmentsEndpoint { get; set; }
+    public string GetDepartmentUsersEndpoint { get; set; }
+    public string SearchDepartmentEndpoint { get; set; }
+
+    [AutoInjectRequest(typeof(IGetUsersDataRequest))]
```

Comment:
It is not necessary but, maybe better to write requests from one service closely